xoverturning/moc.py

```diff
@@ -58,33 +55,98 @@ def calcmoc(
         xarray.DataArray: meridional overturning
     """
 
+    names = define_names(model=model, vertical=vertical)
+
+    if dsgrid is not None:
+        ds = merge_grid_dataset(ds, dsgrid, names)
+
     if remove_hml:
         ucorr, vcorr = substract_hml(ds, umo=umo, vmo=vmo, uhml=uhml, vhml=vhml)
     else:
         ucorr, vcorr = ds[umo], ds[vmo]
 
     if rotate:
-        u_ctr, v_ctr = rotate_velocities_to_geo(ds, ucorr, vcorr)
+        u_ctr, v_ctr = rotate_velocities_to_geo(ds, ucorr, vcorr, names)
     else:
-        u_ctr, v_ctr = interp_to_grid_center(ds, ucorr, vcorr)
+        u_ctr, v_ctr = ucorr, vcorr
+
+    # check vertical dimensions are in the dataarray
+    layer = names["layer"]
+    if layer not in v_ctr.dims:
+        raise ValueError(f"{layer} not found in transport array")
```

Comment:
This line still gets triggered somehow when I run with `rho2` output.

Reply:
``` 
File "/Users/krasting/pkgs/xoverturning/xoverturning/moc.py", line 76, in calcmoc
    raise ValueError(f"{layer} not found in transport array")
ValueError: z_l not found in transport array
```

Reply:
Disregard ... I forgot to set the `vertical` kwarg.

Reply:
it made me realize the readme example was broken (bad copy/paste)